src/main/java/frc/robot/constants/AbstractConstants.java

```diff
@@ -70,17 +76,17 @@ public static class SwerveModuleHardwareIds {
     }
 
     // ========================= Static CONSTANTS ==============================
-    private static final boolean FORCE_GAME_ROBOT_CONSTANTS = false;
+    private static final boolean FORCE_GAME_ROBOT_CONSTANTS = true;
```

Comment:
Should we be doing this in production?

Reply:
Not sure what you mean in production. But this should be `false` for now.